app/Http/Controllers/MembersController.php

```diff
@@ -69,7 +70,27 @@ public function edit(Member $member)
 	# Update member in DB
 	public function update(Member $member, Requests\MemberRequest $request)
 	{
-		$member->update($request->except("roles"));
+		$member->update($request->except(["skills", "roles"]));
+
+		// Update skills
+		$skills = $request->input('skills'); // this is an array with ids of existing skills (as strings!) and string tags of new skills
+
+		$skill_ids = [];
+		foreach ($skills as $skill_id) {
+			if (is_numeric($skill_id)) {
+				$skill = Skill::find((int) $skill_id);
+			} else {
+				$skill = Skill::findOrCreateByTag($skill_id);
+			}
```

Comment:
Why not create a (static) method on the skill entity for this? this way we keep the controller clean and we can reuse the code more easily.